package.json

```diff
@@ -340,7 +340,9 @@
     "onCommand:csharp.downloadDebugger",
     "onCommand:csharp.listProcess",
     "onCommand:csharp.listRemoteProcess",
-    "workspaceContains:project.json"
+    "workspaceContains:project.json",
+    "workspaceContains:*.csproj",
```

Comment:
I'm not 100% sure, but should those be `workspaceContains:**/*.csproj` so that it could be in any folder? It *may* already be handled though